ramls/copycatprofile.json

```diff
@@ -28,13 +28,27 @@
        "type": "string",
        "description": "Specifies where the internalIdentifier is to be embedded in the MARC record, e.g. '999__$i' to place it in field 999, indicator double space (absent) and subfield i; this field only has effect when internalIdentifier is present"
     },
-    "createJobProfileId": {
+    "defaultCreateJobProfileId": {
```

Comment:
No need to make this a breaking change. Keep this property.

Reply:
Thank you @adamdickmeiss, makes sense - we'll keep the properties for createJobProfileId and updateJobProfileId. In case new copycat profile does not have the lists of allowed job profiles, those values will be added to the lists

Reply:
Property keep.